src/types/ErrorTypes.ts

```diff
@@ -0,0 +1,9 @@
+export type ErrorType = 'load' | 'title' | 'add' | 'delete' | 'update';
+
+export enum Errors {
+  load = 'Unable to load todos',
+  title = 'Title should not be empty',
+  add = 'Unable to add a todo',
+  delete = 'Unable to delete a todo',
+  update = 'Unable to update a todo',
+}
```

Comment:
Why you have type and enum for this? Enum is enough, also please use UPPERCASE for enum members, like LOAD = 'Unable to...'